chart/templates/deployment.yaml

```diff
@@ -181,3 +186,17 @@ spec:
         emptyDir: {}
   {{- end }}
 {{- end }}
+{{- if and .Values.customLogos.enabled (or (eq .Values.customLogos.volumeKind "persistentVolumeClaim") (and (eq .Values.customLogos.volumeKind "configMap") (.Values.customLogos.volumeName))) }}
+      - name: custom-logos
+  {{- if (eq .Values.customLogos.volumeKind "persistentVolumeClaim") }}
+        persistentVolumeClaim:
+          claimName: {{ if not .Values.customLogos.volumeName -}}
+      {{ template "rancher.fullname" . }}-custom-logos
+    {{- else -}}
+      {{ .Values.customLogos.volumeName }}
+    {{- end -}}
+  {{- else if and (eq .Values.customLogos.volumeKind "configMap") }}
+        configMap:
+          name: {{ .Values.customLogos.volumeName }}
```

Comment:
this is nil by default, either we need a default here or we should uncomment below

Reply:
Never arriving here if `.Values.customLogos.volumeName` is nil . It's nested to line #189 `if`